types/config.d.ts

```diff
@@ -64,11 +64,11 @@ type ExperimentalConfig = Expand<'all' | Partial<Record<ExperimentalConfigValues
 
 // DarkMode related config
 type DarkModeConfig =
-  /** Use the `media` query strategy. */
+  // Use the `media` query strategy.
   | 'media'
-  /** Use the `class` stategy, which requires a `.dark` class on the `html`. */
+  // Use the `class` stategy, which requires a `.dark` class on the `html`.
   | 'class'
-  /** Use the `class` stategy with a custom class instead of `.dark`. */
+  // Use the `class` stategy with a custom class instead of `.dark`.
```

Comment:
I think that for these it could still be useful to know what each type means. But maybe they don't show up at all 🤔